food-waste-ui/src/components/MarketCard/MarketCard.jsx

```diff
@@ -11,27 +12,29 @@ export default function MarketCard(product) {
   };
 
   return (
-    <div className="product-card">
-      <img
-        className="product-image"
-        src={curProduct.file.url}
-        alt={curProduct.objectId}
-      />
+    <div className="product-card ">
+      <Link to={curProduct.objectId}>
+        <img
+          className="product-image"
+          src={curProduct.file.url}
+          alt={curProduct.objectId}
+        />
+      </Link>
       <div className="description-button-wrapper">
         <div className="product-text-container">
           <div className="product-name">{curProduct.name}</div>
           <div className="product-description">{curProduct.description}</div>
           <div className="product-quantity">{curProduct.quantity}</div>
         </div>
-        <div className="onhold-button">
+        {/* <div className="onhold-button">
           <button onClick={handleOnToggle}>
             {isOnHold ? (
               <Badge color="blue">On Hold</Badge>
             ) : (
               <Badge color="green"> Available</Badge>
             )}
           </button>
-        </div>
+        </div> */}
```

Comment:
nit: clean up the commented out code. I understand that this is nice to have this as a note, but we would like to have each diff represent a production quality change